examples/esi_example_advanced_error_handling/Cargo.toml

```
@@ -9,6 +9,5 @@ publish = false
 [dependencies]
 fastly = "^0.10"
 esi = { path = "../../esi" }
-quick-xml = "0.36.0"
 log = "^0.4"
-env_logger = "^0.11" # 0.10.0 requires nightly
+env_logger = "^0.11"         # 0.10.0 requires nightly
```

Nothing to do with your changes but this comment isn't relevant anymore.

```suggestion
env_logger = "^0.11"
```

---

yes, I already did that before, but missed to clean this up when rebased. good catch!